index.html

```diff
@@ -1,16 +1,99 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
 
```

Comment:
You missed the viewport meta tag. It might look like media queries still work when you're testing it in your browser but if you test it in an actual phone or tablet, it will only show your desktop view. 😄

Reply:
That's why it wasn't working on the device emulator in devtools....